src/Controls/src/Core/Platform/AlertManager/AlertManager.Windows.cs

```diff
@@ -16,17 +16,17 @@ internal void Subscribe(Window window)
 		{
 			var platformWindow = window.MauiContext.GetPlatformWindow();
 
-			if (Subscriptions.Any(s => s.Window == platformWindow))
+			if (Subscriptions.Any(s => s.PlatformView == platformWindow))
 				return;
 
-			Subscriptions.Add(new AlertRequestHelper(platformWindow, window.MauiContext));
+			Subscriptions.Add(new AlertRequestHelper(window, platformWindow));
 		}
 
 		internal void Unsubscribe(Window window)
 		{
 			var platformWindow = window.MauiContext.GetPlatformWindow();
 
-			var toRemove = Subscriptions.Where(s => s.Window == platformWindow).ToList();
+			var toRemove = Subscriptions.Where(s => s.PlatformView == platformWindow).ToList();
```

Comment:
@PureWeen  I though that the use of LINQ was limited, should we remove it from here?